src/test/java/org/jabref/logic/importer/fileformat/BibtexParserTest.java

```diff
@@ -2238,4 +2240,42 @@ void parseInvalidBibDeskFilesResultsInWarnings() throws IOException {
 
         assertEquals(List.of(firstEntry, secondEntry), result.getDatabase().getEntries());
     }
+
+    @Test
+    void parseCommentToJson() {
+        String entries =
+                """
+                    @Comment{jabref-meta-0.1.0
+                        {
+                            "saveActions" :
+                            {
+                                "state": true,
+                                "date": ["normalize_date", "action2"],
+                                "pages" : ["normalize_page_numbers"],
+                                "month" : ["normalize_month"]
+                            }
+                        }
+                """;
+        BibtexParser parser = new BibtexParser(importFormatPreferences);
+        Optional<JsonObject> actualJson = parser.parseCommentToJson(entries);
+        assertEquals(actualJson, Optional.of(getExpectedJson()));
+    }
+
+    private JsonObject getExpectedJson() {
+        JsonObject saveActions = new JsonObject();
+        saveActions.addProperty("state", true);
+        JsonArray dateArray = new JsonArray();
+        dateArray.add("normalize_date");
+        dateArray.add("action2");
+        saveActions.add("date", dateArray);
+        JsonArray pagesArray = new JsonArray();
+        pagesArray.add("normalize_page_numbers");
+        saveActions.add("pages", pagesArray);
+        JsonArray monthArray = new JsonArray();
+        monthArray.add("normalize_month");
+        saveActions.add("month", monthArray);
+        JsonObject expectedJson = new JsonObject();
+        expectedJson.add("saveActions", saveActions);
+        return expectedJson;
```

Comment:
This reads very ugly.

According to https://stackoverflow.com/a/4696873/873282 it can be easier

Can you try with `var`?

```java
var expected = new Object() {
...
```

Another thought: Just simplify the contained object.

you are testing the JSON reading - then put a **simple** JSON inside. Test one thing in one test :-)

Reason: The searialization of `saveActions` will probalby changed. Especially `state` will be called `enabled` for sure (because `state` has a different meaning)

Reply:
I tried using the var expected = new Object() {} style, but since Gson's JsonObject is declared as final, it throws an exception like Cannot inherit from final 'com.google.gson.JsonObject' when attempting this code:
```
var expectedJson = new JsonObject() {{
    add("saveActions", new JsonObject() {{
        addProperty("state", true);
    }});
}};
```
P.S.: I had a bike accident and got injured, but I’ve finally fully recovered. Apologies for disappearing for such a long time!"

Reply:
Im sorry to hear, I hope it wasn't too bad and you are well again. Don't worry about time, better slow and steady instead of rushing things in bad quality and never appearing again.